windows.js

```diff
@@ -14,11 +14,12 @@ const rubyInstallerVersions = require('./windows-versions.json')
 const drive = common.drive
 
 const msys2BasePath = 'C:\\msys64'
+const msys2GCCReleaseURI  = 'https://github.com/ruby/setup-msys2-gcc/releases/download'
 
-// needed for 2.0-2.3, and mswin, cert file used by Git for Windows
+// needed for Ruby 2.0 thru 2.3, and mswin, cert file used by Git for Windows
```

Comment:
I'm not a fan of `2.0 thru 2.3` (looks like badly spelled C code), it could be `through` but that's long and not easy to read.
Can you use use `for Ruby 2.0-2.3`, same below?